configs/vax/OH.yaml

```diff
@@ -0,0 +1,18 @@
+state: OH
+
+links:
+- name: vaccine
+url: https://coronavirus.ohio.gov/wps/portal/gov/covid-19/dashboards/covid-19-vaccine/covid-19-vaccination-dashboard
```

Comment:
hi! i think these would also need to be indented (url, overseerScript, message)

Reply:
yes! will add, was typing this while on the dq shift zoom 🤦